recipe/meta.yaml

```diff
@@ -56,3 +56,4 @@ extra:
   recipe-maintainers:
     - ocefpaf
     - kwilcox
+    - xylar
```

Comment:
:+1:  this will give you the power to self-merge updates like this one in the future.

PS: I used to get e-mails with the new releases, not sure why that is no longer happening :confused:

Reply:
Hmm, I heard about this release directly from @czender.  Maybe the email just hasn't gone out yet?

Reply:
Awesome! If you have such a close relationship with the releases it only makes more sense o get you in the team :smile: